components/docs-chef-io/content/automate/bootstrap_on_prem_builder.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Bootstrap Chef Habitat Builder On-prem"
+
+date = 2019-11-19T14:10:15-08:00
+
+draft = false
+
+[menu]
+  [menu.automate]
+    title = "Bootstrap Chef Habitat Builder On-prem"
+    parent = "automate/getting_started"
+    identifier = "automate/getting_started/bootstrap_on_prem_builder.md Bootstrap Chef Habitat Builder On-prem"
+    weight = 55
```

Comment:
I set the weight to something that would make the bootstrap page show up in the ToC after the page on installing on-prem Builder; there's nothing special about the value I picked.